static/style/stylesheet.css

```diff
@@ -60,8 +70,9 @@
     border: 0;
 }
 .tfclear{
-    clear:both;
+    float: left;
```

Comment:
There are more problems. Purpouse of this was not to rewrite code or clean it up, but rather use responsive actions instead of pixel determined locations. If cleanup is needed at this stage, it should be made as seperate task.